app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+class Animal:
+    alive = []
```

Comment:
The variable name `alive` is not very descriptive. Consider renaming it to something more meaningful. For example, `living_animals` or `animals_alive` would be better choices.